feat: add Dart v1.0 developer preview docs (#8844)
supabase/supabase: apps/reference/_supabase_dart/generated/storage-from-upload.mdx

    </span>
  </h4>
  <div class="method-list-item-description">
  
<ul className="method-list-group">
  <h5 class="method-list-title method-list-title-isChild expanded">Properties</h5>

<li className="method-list-item">
  <h4 className="method-list-item-label">
    <span className="method-list-item-label-name">
      cacheControl
    </span>
    <span className="method-list-item-label-badge false">
      optional
    </span>
    <span className="method-list-item-validation">
      <code>string</code>
    </span>
  </h4>
  <div class="method-list-item-description">

The Cache-Control: max-age=[seconds] seconds value.

  </div>
  
</li>

<li className="method-list-item">
  <h4 className="method-list-item-label">
    <span className="method-list-item-label-name">
      contentType
    </span>
    <span className="method-list-item-label-badge false">
      optional
    </span>
    <span className="method-list-item-validation">
      <code>string</code>
    </span>
  </h4>
  <div class="method-list-item-description">

> The Content-Type header value. The SDK will read the content type and set the header for you by default.

Reviewer: ```suggestion
The Content-Type header value. By default, the SDK reads the content type and sets the header.
```

  <div class="method-list-item-description">

HTTP headers.
`cacheControl`: string, the `Cache-Control: max-age=<seconds>` seconds value. <br />

> `contentType`: string, the `Content-Type` header value. The SDK will read the content type and set the header for you by default.<br />

Reviewer: ```suggestion
`contentType`: string, the `Content-Type` header value. By default, the SDK reads the content type and sets the header.<br />
```

> `contentType`: string, the `Content-Type` header value. The SDK will read the content type and set the header for you by default.<br />

Reviewer: The formatting here is kind of hard to parse:
![Screen Shot 2022-09-08 at 12 23 26 PM](https://user-images.githubusercontent.com/7026076/189208583-04208894-859c-4d70-8fbd-7fd7e8060d31.png)

Can we make this a "properties" table (e.g., [signUp()](https://supabase.com/docs/reference/javascript/auth-signup#parameters))?
![Screen Shot 2022-09-08 at 12 24 44 PM](https://user-images.githubusercontent.com/7026076/189208697-5470b056-be91-4cc0-b38a-ee48b501f64d.png)

Author: Sounds good! Let me do this now!

Author: Done!
